Makefile

```
@@ -93,5 +93,5 @@ build-resources: docker-all helm-charts cnudie
 
 .PHONY: build-int-test-image
 build-int-test-image:
-	@docker build integration-test/docker -t eu.gcr.io/gardener-project/landscaper-service/integration-test:1.19.10-alpine3.18
-	@docker push eu.gcr.io/gardener-project/landscaper-service/integration-test:1.19.10-alpine3.18
+	@docker buildx build --platform linux/amd64 integration-test/docker -t eu.gcr.io/gardener-project/landscaper-service/integration-test:1.20.6-alpine3.18
+	@docker push eu.gcr.io/gardener-project/landscaper-service/integration-test:1.20.6-alpine3.18
```

Can we change this to:

```suggestion
	@docker buildx build --platform linux/amd64 integration-test/docker -t eu.gcr.io/gardener-project/landscaper-service/integration-test:1.20.6-alpine3.18 --push
```

Otherwise this locally fails with:
```
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
```